### DISCUSSION TITLE
dateColumn: Allow rowData to be a (parseable) string

### DISCUSSION THREAD

**Discussion by ajboni** [nick-keller/react-datasheet-grid#148] [Feature requests]
https://github.com/Equify/react-datasheet-grid/blob/4a4c955bf3d46adb7dc75577e62d3ed880705e01/src/columns/dateColumn.tsx#L30-L32

Coming from a serialized json dataset where the data for the date column is a string this will throw an error trying to convert to isoString. 
To avoid the burden of manually creating a date object for every row on initialization, maybe datecolumn could tryParse for a datetime string.

Or maybe,  just  directly use strings?

**Comment by nick-keller** [comment]
Hey @ajboni ,
The built-in date column is not really meant to be used "as is" in production, it is mostly there to be used as an example. I would highly suggest implementing your own column using the date component of your component library.

As it is a fairly simple column, feel free to suggest a pull request to enhance it, can can move forward from there ;)

**Reply by ajboni** [reply]
Makes sense! I want to contribute eventually and this sounds like a good first issue... only need to find some time!

**Comment by nick-keller** [comment]
Hey @ajboni ,
This is now available in 4.8.0 as `isoDateColumn`
